internal/server/evaluation/evaluator.go

```diff
@@ -12,26 +12,20 @@ import (
 	"go.flipt.io/flipt/internal/server/metrics"
 	"go.flipt.io/flipt/internal/storage"
 	"go.flipt.io/flipt/rpc/flipt"
+	rpcEvaluation "go.flipt.io/flipt/rpc/flipt/evaluation"
```

Comment:
💅 tend to just lowercase package names:
```suggestion
	rpcevaluation "go.flipt.io/flipt/rpc/flipt/evaluation"
```